README.md

```diff
@@ -35,7 +35,10 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-262-orange.svg?style=flat-square)](#contributors-)
+feat/add-richard
 
+
+main
```

Comment:
- After solving the merge conflicts and run the command to generate profile, the badge should increase. But this hasn't increased.
- These lines of code are not supposed to be here. Please remove them.